sdk/servicebus/perf-tests/service-bus/test/subscribe.spec.ts

```diff
@@ -69,6 +69,7 @@ export class SubscribeTest extends EventPerfTest<ReceiverOptions> {
   setup() {
     this.subscriber = this.receiver.subscribe(
       {
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
         processMessage: async (_message: ServiceBusReceivedMessage) => {
```

Comment:
nit: could the parameter be omitted?
```suggestion
        processMessage: async () => {
```

Reply:
I read the message wrong, this is actually a warning. I will undo the change to this file.